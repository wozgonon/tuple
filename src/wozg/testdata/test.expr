a(b, c, d)
a(b(c))
a(1, 2, 3)
1+2

acos(cos(PI))==PI

atan(1,1) == 0.7853981633974483

PI+PI==2*PI

()

(1)

(1+1)

(1+PI)

func abc {
	 1+2 
	 3*9
}
