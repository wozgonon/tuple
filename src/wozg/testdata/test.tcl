hello { ab { 1 2 3 } }

1 2
abc "def" "tab\ttab\ttab" NaN Inf {1-aa} 123 {} {+ 1 2} .123 456. 123.456 "世界"

Check if spaces needed{aa==bb}between braces{ab{def{efg}}} 2 3


ls a whole line

hello {
    This is a line of text.

    And another.
}

