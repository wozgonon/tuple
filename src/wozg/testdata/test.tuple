a(b, c, d)
a(b(c))
a(1, 2, 3)
+(1, 2)
