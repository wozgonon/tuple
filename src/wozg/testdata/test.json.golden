{
    "menu" :
    {
            "id" : "file",
            "value" : "File",
            "a" :
        [
          1,
          2,
          3
        ],
            "popup" :
        {
                    "menuitem" :
            [
              {
                                "value" : "New",
                                "onclick" : "CreateNewDoc()"
              },
              {
                                "value" : "Open",
                                "onclick" : "OpenDoc()"
              },
              {
                                "value" : "Close",
                                "onclick" : "CloseDoc()"
              }
            ]
        }
    }
}
